README.md

```diff
@@ -13,53 +13,221 @@ The toolkit grew out of a previous project called [Warcbase](https://github.com/
 
+```shell
+$ spark-shell --help
+
+  --jars JARS                 Comma-separated list of jars to include on the driver
+                              and executor classpaths.
+  --packages                  Comma-separated list of maven coordinates of jars to include
+                              on the driver and executor classpaths. Will search the local
+                              maven repo, then maven central and any additional remote
+                              repositories given by --repositories. The format for the
+                              coordinates should be groupId:artifactId:version.
 ```
-$ spark-shell --packages "io.archivesunleashed:aut:0.18.0"
+
+#### As a package
+
+Release version:
+
+```shell
+$ spark-shell --packages "io.archivesunleashed:aut:0.18.1"
 ```
 
-### A little less easy
+HEAD (built locally):
+
+```shell
+$ spark-shell --packages "io.archivesunleashed:aut:0.18.2-SNAPSHOT"
+```
 
-You can download the [latest release here](https://github.com/archivesunleashed/aut/releases) and include it like so:
+#### With an UberJar
 
+Release version:
+
+```shell
+$ spark-shell --jars /path/to/aut-0.18.1-fatjar.jar
+```
+
+HEAD (built locally):
+
+```shell
+$ spark-shell --jars /path/to/aut-0.18.2-SNAPSHOT-fatjar.jar
+```
+
+### Archives Unleashed Toolkit with PySpark
+
+To run PySpark with the Archives Unleashed Toolkit loaded, you will need to provide PySpark with the Java/Scala package, and the Python bindings. The Java/Scala packages can be provided with `--packages` or `--jars` as described above. The Python bindings can be [downloaded](https://github.com/archivesunleashed/aut/releases/download/aut-0.18.1/aut-0.18.1.zip), or [built locally](#Introduction) (the zip file will be found in the `target` directory.
+
+#### As a package
+
+Release version:
+
+```shell
+$ export PYSPARK_PYTHON=/path/to/python; export PYSPARK_DRIVER_PYTHON=/path/to/python; /path/to/spark/bin/pyspark --py-files aut-0.18.1.zip --packages "io.archivesunleashed:aut:0.18.1"
+```
+
+HEAD (built locally):
+
+```shell
+$ export PYSPARK_PYTHON=/path/to/python; export PYSPARK_DRIVER_PYTHON=/path/to/python; /path/to/spark/bin/pyspark --py-files /home/nruest/Projects/au/aut/target/aut.zip --packages "io.archivesunleashed:aut:0.18.2-SNAPSHOT"
+```
+
+#### With an UberJar
+
+Release version:
+
+```shell
+$ export PYSPARK_PYTHON=/path/to/python; export PYSPARK_DRIVER_PYTHON=/path/to/python; /path/to/spark/bin/pyspark --py-files aut-0.18.1.zip --jars /path/to/aut-0.18.1-fatjar.jar
 ```
-$ spark-shell --jars /path/to/aut-0.18.0-fatjar.jar"
+
+HEAD (built locally):
+
+```shell
+$ export PYSPARK_PYTHON=/path/to/python; export PYSPARK_DRIVER_PYTHON=/path/to/python; /path/to/spark/bin/pyspark --py-files /home/nruest/Projects/au/aut/target/aut.zip --jars /path/to/aut-0.18.2-SNAPSHOT-fatjar.jar
 ```
 
-### Even less easy
+### Archives Unleashed Toolkit with Jupyter
+
```

Comment:
add a note that they'll need to have Jupyter Notebooks installed?  Just link out to https://jupyter.org/install?